pkg/util/validation/limits.go

```diff
@@ -435,6 +436,11 @@ func (o *Overrides) AcceptHASamples(userID string) bool {
 	return o.getOverridesForUser(userID).AcceptHASamples
 }
 
+// ServiceOverloadStatusCodeOnRateLimitEnabled return whether the distributor uses error code 529 instead of 429 when the rate limit is exceeded.
```

Comment:
```suggestion
// ServiceOverloadStatusCodeOnRateLimitEnabled return whether the distributor uses status code 529 instead of 429 when the rate limit is exceeded.
```